src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  setModalTodo: React.Dispatch<React.SetStateAction<Todo | undefined>>,
+  modalTodo: Todo | undefined,
```

Comment:
You can make this field optional by `?` and don't use `| undefined`

Like this `modalTodo?: Todo`